doc/Makefile

```diff
@@ -131,7 +131,8 @@ MANPAGES_3_MD += librpmem/rpmem_persist.3.md
 MANPAGES_1_MD += rpmemd/rpmemd.1.md
 MANPAGES_3_DUMMY += rpmem_open.3 rpmem_set_attr.3 rpmem_close.3 \
 		    rpmem_read.3 rpmem_remove.3 rpmem_check_version.3 \
-		    rpmem_errormsg.3 rpmem_deep_persist.3
+		    rpmem_errormsg.3 rpmem_deep_persist.3 rpmem_flush.3 \
```

Comment:
You should also add aliases to doc/generated

Reply:
pmem/pmdk#3568